contracts/BatchAuction.sol

```diff
@@ -0,0 +1,24 @@
+pragma solidity ^0.5.0;
+
+
+contract BatchAuction {
+
+    uint16 public maxAccountNumber = 10000;
```

Comment:
Let's use 100 for now? A snark for 10k still requires a lot of optimization.

Reply:
Also let's use the constant keyword here: https://solidity.readthedocs.io/en/v0.4.24/contracts.html#constant-state-variables

Reply:
I can use 100 for now, but then should I reduce this to uint8?